.yamllint

```diff
@@ -0,0 +1,15 @@
+extends: default
+ignore: |
+  .tox
+
+rules:
+  braces:
+    max-spaces-inside: 1
+    level: error
+  brackets:
+    max-spaces-inside: 1
+    level: error
+  document-start: disable
+  line-length: disable
```

Comment:
Maybe have a sane default like 160 or 200?

Reply:
I think 160 is default on ansible-lint, so ok, we can always revise them later.

Reply:
Correct, 160 is the default for Ansible.